README.md

```diff
@@ -565,14 +565,53 @@ assert.Len(t, arr, 3)
 ```go
 ❌
 assert.Less(t, a, 0)
+assert.Less(t, a, int(0))
+assert.Less(t, a, int8(0))
+assert.Less(t, a, int16(0))
+assert.Less(t, a, int32(0))
+assert.Less(t, a, int64(0))
+assert.Less(t, a, uint(0))
+assert.Less(t, a, uint8(0))
+assert.Less(t, a, uint16(0))
+assert.Less(t, a, uint32(0))
+assert.Less(t, a, uint64(0))
 assert.Greater(t, 0, a)
+assert.Greater(t, int(0), a)
+assert.Greater(t, int8(0), a)
+assert.Greater(t, int16(0), a)
+assert.Greater(t, int32(0), a)
+assert.Greater(t, int64(0), a)
+assert.Greater(t, uint(0), a)
+assert.Greater(t, uint8(0), a)
+assert.Greater(t, uint16(0), a)
+assert.Greater(t, uint32(0), a)
+assert.Greater(t, uint64(0), a)
 assert.True(t, a < 0)
```

Comment:
why no changes around expression in True/False?

this is relevant for this too, isn't it?

Reply:
I think I suggested it in another package and you would have said, we had no examples of such usage.

I might have misunderstood also.

But I will add then then, sure